### DISCUSSION TITLE
[luxon] DateObjectUnits interface missing some properties

### DISCUSSION THREAD

**Discussion by utherpally** [DefinitelyTyped/DefinitelyTyped#65962] [Issues with a @types package]
The properties of DateObjectUnits interface should be generated from https://github.com/moment/luxon/blob/3.3.0/src/datetime.js#L306-L329

Currently, `<Datetime>.set({quarter: 3})` gives me a type error: 

```
Type '{ quarter: number; }' is not assignable to type 'DateObjectUnits'.
  Object literal may only specify known properties, and 'quarter' does not exist in type 'DateObjectUnits'.
```

*Updated: fix typo*

[a comment by an automated account is not shown]

**Comment by CarsonF** [comment]
Yes that's because `.set({quarter: 3})` is not supported and does nothing at runtime.
